server/configs/webapps/embedded/config/application.properties

```diff
@@ -0,0 +1,163 @@
+server.port=8080
```

Comment:
What's the difference between this and `server/configs/application.properties`? Can we capture that in a comment in one or both files?

Reply:
Everything in `server/configs/webapps/embedded` will be included in embedded distributions. I was going to to a follow-up PR with updates to the `application.properties` and `README` files